test/skyux.spec.js

```diff
@@ -30,18 +30,29 @@ describe('skyux CLI', () => {
       }
     });
 
-    const cli = require('../index');
+    const cli = mock.reRequire('../index');
     cli({ _: [''], v: true });
     expect(called).toEqual(true);
   });
 
+  it('should accept known command help', () => {
```

Comment:
Question: Should we stop the propagation of "known" commands? For example, if I type `skyux help`, this command gets passed to all other `skyux-builder-*` modules as well. Just curious what your thoughts are on "locking down" top-level commands:

```
skyux help
warn: Found matching module without exposed runCommand: @blackbaud/skyux-builder-plugin-stache
info: Passing command to @blackbaud/skyux-builder-stache-search
info: Passing command to @blackbaud/skyux-builder
```

Reply:
Hmm.  That was how the previous behavior worked.  My thought with this is that other commands to could choose to still answer the command.  `help` and `version` being good examples where I envisioned all relevant modules responding.  A few possible solutions...

1.  Don't display an message if modules match the name but don't have a runCommand.
2.  Made the glob pattern exclude some results.  Not entirely sure what that would even be.
3.  Do as you proposed and have those top level commands not pass down.  I think the problem here would be that all commands are still going to get passed down to stache and show that message.

Sounds like I've talked myself into number 1 being the only viable solution.  What do you think?

Reply:
I can see your point, RE: it'd be good to have `skyux help` pass down to the other modules. I think we're okay to leave it as-is, as the warning could be useful for developers. Maybe we simply change the `warn` to an `info`, so consumers don't think something's busted?

Reply:
I'm just wondering if I show show a message at all.  I wasn't thinking about Stache sites.  For every one of them, for every command we run on it (serve, build, test, etc), that message would be logged.

Reply:
True; probably wouldn't hurt to remove it then.